system-parachains/collectives/collectives-polkadot/src/secretary/mod.rs

```diff
@@ -0,0 +1,350 @@
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type Scheduler = Scheduler;
+	type Currency = Balances;
+	// Secretary collective can submit proposals.
+	type SubmitOrigin = pallet_ranked_collective::EnsureMember<
+		Runtime,
+		SecretaryCollectiveInstance,
+		{ ranks::SECRETARY },
+	>;
+	// Referandum can be cancled by any of:
+	// - Root;
+	// - a member of the Secretary Program.
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	type CancelOrigin = OpenGovOrSecretary;
+	// Referandum can be killed by any of:
+	// - Root;
+	// - a member of the Secretary Program.
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	type KillOrigin = OpenGovOrSecretary;
+	type Slash = ToParentTreasury<PolkadotTreasuryAccount, LocationToAccountId, Runtime>;
+	type Votes = pallet_ranked_collective::Votes;
+	type Tally = pallet_ranked_collective::TallyOf<Runtime, SecretaryCollectiveInstance>;
+	type SubmissionDeposit = ConstU128<0>;
+	type MaxQueued = ConstU32<100>;
+	type UndecidingTimeout = ConstU32<{ 7 * DAYS }>;
+	type AlarmInterval = ConstU32<1>;
+	type Tracks = tracks::TracksInfo;
+	type Preimages = Preimage;
+}
+
+impl pallet_ranked_collective::Config<SecretaryCollectiveInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_ranked_collective_secretary_collective::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	// Promotions and inductions should be done through the [`crate::SecretaryCore`] pallet instance
+	// instance.
+	type PromoteOrigin = frame_system::EnsureNever<pallet_ranked_collective::Rank>;
+	#[cfg(feature = "runtime-benchmarks")]
+	// The maximum value of `u16` set as a success value for the root to ensure the benchmarks will
+	// pass.
+	type PromoteOrigin = EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>;
+	// Demotion is by any of:
+	// - Root can demote arbitrarily.
+	// - a single member of the Fellowship Program (DAN III);
+	type DemoteOrigin = EitherOf<
+		EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>,
+		MapSuccess<
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				FellowshipCollectiveInstance,
+				{ DAN_3 },
+			>,
+			Replace<ConstU16<2>>,
+		>,
+	>;
+	// Exchange is by any of:
+	// - Root can exchange arbitrarily.
+	// - the Secretary Origin.
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	type ExchangeOrigin = OpenGovOrSecretary;
+	type Polls = SecretaryReferenda;
+	type MinRankOfClass = Identity;
+	type MemberSwappedHandler = (crate::SecretaryCore, crate::SecretarySalary);
+	type VoteWeight = pallet_ranked_collective::Geometric;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkSetup = (crate::SecretaryCore, crate::SecretarySalary);
+}
+
+pub type SecretaryCoreInstance = pallet_core_fellowship::Instance3;
+
+impl pallet_core_fellowship::Config<SecretaryCoreInstance> for Runtime {
```

Comment:
Yeah, Sounds good.

Reply:
Also like to point out that without the core pallet there is no time contraint, especially when it come to paypot dispersion.

Reply:
This would require implementing the upgraded ranked collective pallet, with the inclusion of [`AddOrigin`](https://github.com/paritytech/polkadot-sdk/blob/fc6c31829fc2e24e11a02b6a2adec27bc5d8918f/substrate/frame/ranked-collective/src/lib.rs#L396) and [`RemoveOrigin`](https://github.com/paritytech/polkadot-sdk/blob/fc6c31829fc2e24e11a02b6a2adec27bc5d8918f/substrate/frame/ranked-collective/src/lib.rs#L401).

Reply:
The core pallet is also used for `MemberSwappedHandler` and I think we should leave the core pallet in.

The SecretaryCollective is bound to have more than one member in the coming months as developments progress.